### ISSUE TITLE
How to show gather telemetry 

### ISSUE THREAD

**Issue by tarekradi** [jenningsloy318/redfish_exporter#38]
Hi, i have a successful connection between Prometheus and the redfish exporter. I am able to show things such as the following.
![image](https://user-images.githubusercontent.com/13227624/141881093-a4a7c613-1417-41b2-9c26-f46ccd6d95e6.png)

What is the Prometheus search query that i can use to load up the PowerConsumedWatts telemetry that would appear when i make the following direct redfish call?
![image](https://user-images.githubusercontent.com/13227624/141881212-db94bd22-0350-4669-a9a8-d18affc7c149.png)

Or load up SEL entries such as the following?
![image](https://user-images.githubusercontent.com/13227624/141881391-65dd1db7-d4d8-476c-b3ed-4ea56fbe9e22.png)

Thank you.

**Comment by jenningsloy318** [comment]
don't understand your point, do you mean that you need watts metrics ? if yes, it is  not implemented yet. and will not update it in near future.

**Comment by tarekradi** [comment]
I'm saying that the system can respond to several redfish commands that i don't know how to view in Prometheus. Here are some examples of redfish RESTful commands that the systems can reply to but i don't know how to query from inside Prometheus

http://10.219.111.126/redfish/v1/Chassis/RackMount/Baseboard/Power
http://10.219.111.126/redfish/v1/Systems/............/LogServices/SEL/Entries

And many others. How can i view the results of these calls in Prometheus and track them?

**Comment by jenningsloy318** [comment]
view these response in prometheus requires that it is implemented in some exporter that prometheus, here for the redfish-exporter, these functions are not implemented, so you can't view them

**Comment by tarekradi** [comment]
Can you summarize the changes needed in the code so that they can be exported?

**Comment by tarekradi** [comment]
friendly ping. What changes do i need to make in the code to start exposing the redfish results of the following commands?

http://10.219.111.126/redfish/v1/Chassis/RackMount/Baseboard/Power
http://10.219.111.126/redfish/v1/Systems/............/LogServices/SEL/Entries

**Comment by Hydrapozza** [comment]
Hello tarekradi,
Do you found a solution to add new commands to the code? I also need to add some.
